app/controllers/people_controller.rb

```diff
@@ -40,10 +40,13 @@ def person
     @person ||= Person.find(params[:person_id])
   end
 
-  def export
-    odt_file = Odt::Cv.new(entry).export
+  # rubocop:disable Metrics/LineLength
+  def export(anon = params[:anon].presence || 'false')
+    odt_file = Odt::Cv.new(entry).export(anon != 'false')
```

Comment:
Refactor this so you pass the params to the constructor... This way it's also expendable and you don't need to do these weird checks.
Also, if you do set a default value for a method, avoid using operators and other objects. It should be a fallback, what you're doing is assigning a variable.